src/test/common/steps/address-lookup-steps.js

```diff
@@ -1,15 +1,25 @@
 /* eslint-disable no-unused-expressions */
-const { When, Then } = require('cucumber')
+const { Given, When, Then } = require('cucumber')
 const { expect } = require('chai')
 const Promise = require('bluebird')
 
 const pages = require('./pages')
-const { enterPostcodeAndSubmit } = require('./common-steps')
+const { enterDetailsUpToPage, DEFAULT_ACTION_OPTIONS, STEP_PAGE_ACTIONS } = require('./navigation')
+const { enterPostcodeAndSubmit, selectFirstAddress } = require('./common-steps')
 const { setupPostcodeLookupWithNoResults, setupPostcodeLookupWithResults } = require('../wiremock')
 const { POSTCODE } = require('./constants')
 
 const POSTCODE_WITH_NO_RESULTS = 'BS11AA'
 
+const stepActionsWithoutManualAddress = (stepPageActions) => stepPageActions.filter(pageAction => pageAction.page(pages) !== pages.manualAddress)
+
+Given(/^I have entered my details up to the check details page and selected an address$/, async function () {
+  const actionOptions = { ...DEFAULT_ACTION_OPTIONS, selectAddress: true }
+  // manual address page is skipped when the user selects an address
+  const stepActions = stepActionsWithoutManualAddress(STEP_PAGE_ACTIONS)
```

Comment:
I think that these step actions should be somewhere more central so that they can be reused if required, possibly along with `step-page-actions.js` or in the same folder so we can collect all the variants together.

Reply:
We discussed this but decided against it as they're only used in this one place. They're specific to address lookup.